src/Altinn.App.Core/Features/Signing/SigningService.cs

```diff
@@ -196,6 +201,33 @@ AltinnSignatureConfiguration signatureConfiguration
         return signeeContexts;
     }
 
+    // <inheritdoc />
+    public async Task<List<OrganisationSignee>> GetAuthorizedOrganisations(
+        IInstanceDataAccessor instanceDataAccessor,
+        AltinnSignatureConfiguration signatureConfiguration,
+        int userId
+    )
+    {
+        List<SigneeContext> signeeContexts = await TryDownLoadSigneeContexts(
+            instanceDataAccessor,
+            signatureConfiguration
+        );
+
+        List<OrganisationSignee> authorizedOrganisations = [];
+        List<OrganisationSignee> orgSignees = [.. signeeContexts.Select(x => x.Signee).OfType<OrganisationSignee>()];
+
+        foreach (OrganisationSignee organisationSignee in orgSignees)
+        {
+            List<Role> roles = await authorizationClient.GetRoles(userId, organisationSignee.OrgParty.PartyId);
+            if (roles.Count != 0)
```

Comment:
Superpirk for lesbarhet
```suggestion
            var hasKeyRoleInOrg = roles.Count != 0;
            if (hasKeyRoleInOrg)
```

Reply:
Hva er en nøkkelrolle? Landet vi ikke på at siden parallellsigneringsrollen ikke finnes i altinn 3 så ser vi etter organisationsignees der bruker har en hvilken som helst rolle - roller som i autorisasjonsspråket kalles "decisionRole"